pytorch_lightning/trainer/supporters.py

```diff
@@ -90,3 +92,60 @@ def accumulate(self, x):
 
     def mean(self):
         return self.total / self.num_values
+
+
+class PredictionCollection(object):
+
+    def __init__(self, global_rank: int, world_size: int):
```

Comment:
is there no way to extract these arguments automatically? So that we don't have to pass them manually?